app/scripts/directives/oscPersistentVolumeClaim.js

```diff
@@ -26,7 +27,26 @@ angular.module("openshiftConsole")
         }];
         scope.claim.selectedLabels = [];
         DataService.list({group: 'storage.k8s.io', resource: 'storageclasses'}, {}, function(storageClasses) {
-          scope.storageClasses = storageClasses.by('metadata.name');
+          scope.storageClasses = _.map(storageClasses.by('metadata.name'));
+          var defaultClassAnnotation = "storageclass.beta.kubernetes.io/is-default-class";
+          _.each(scope.storageClasses, function(sclass){
+            if ("annotations" in sclass.metadata && defaultClassAnnotation in sclass.metadata.annotations && sclass.metadata.annotations[defaultClassAnnotation] === "true") {
+              scope.defaultStorageClass = sclass.metadata.name;
+            }
+          });
+          if (scope.defaultStorageClass === "")  { //if there is no default, set a no storage class option
```

Comment:
> If we know which storage class is the default I would rather see us pre-select that option and have an indicator like (Default) next to that name.

I'm fine with that. I wasn't sure if there is ever a reason to leave it blank instead of explicitly setting the default.